app/src/main/res/drawable/ic_quest_bus_stop_name.xml

```diff
@@ -0,0 +1,170 @@
+												android:fillColor="#99aab5" />
+											</group>
+											<path
+												android:pathData="M-684.9999 -43.90647l0 19.32227c2.5609 1.48206 5.2305 2.7926 8 3.91406l0 -23.23633z"
+												android:fillColor="#666666" />
+											<group
+												android:translateY="4">
+												<path
+													android:pathData="M-660.99986 -52.107489l0 4.861359c0 1.850171 -1.48948 3.339659 -3.33965 3.339659l-33.32071 0c-1.85017 0 -3.33966 -1.489488 -3.33966 -3.339659l0 -4.861359z"
+													android:fillColor="#000000"
+													android:strokeLineJoin="round"
+													android:strokeLineCap="round"
+													android:strokeMiterLimit="4"
+													android:fillAlpha="0.2" />
+												<path
+													android:pathData="M-696.99988 -101.90647l32.00002 0c2.216 0 4 1.784 4 4l0 46.000013c0 2.216 -1.784 4.000001 -4 4.000001l-32.00002 0c-2.216 0 -4 -1.784001 -4 -4.000001l0 -46.000013c0 -2.216 1.784 -4 4 -4z"
+													android:fillColor="#ffffff"
+													android:strokeLineJoin="round"
+													android:strokeLineCap="round"
+													android:strokeMiterLimit="4" />
+												<group
+													android:translateY="2.90625">
+													<path
+														android:pathData="M-665.99204 -84.84156a15.000022 15.062499 0 0 1 -15.00003 15.062499 15.000022 15.062499 0 0 1 -15.00002 -15.062499 15.000022 15.062499 0 0 1 15.00002 -15.062499 15.000022 15.062499 0 0 1 15.00003 15.062499z"
+														android:fillColor="#495aad"
+														android:strokeLineJoin="round"
+														android:strokeLineCap="round"
+														android:strokeMiterLimit="4" />
+													<group
+														android:translateX="-1613"
+														android:translateY="-661.9065">
+														<path
+															android:pathData="M926 567.07192c-1.108 0 -2 0.892 -2 2L924 583c0 0.74107 0.4042 1.37905 1 1.72461l0 3.27539 4 0 0 -3 6 0 0 3 4 0 0 -3.27539c0.5958 -0.34556 1 -0.98354 1 -1.72461l0 -13.92808c0 -1.108 -0.892 -2 -2 -2zm3 2.92808l6 0c1.108 0 2.01718 0.89213 2 2l0 5 -10 0 0 -5c0 -1.108 0.892 -2 2 -2zm-0.95581 9.04699c1.10457 0 2 0.89543 2 2 0 1.10457 -0.89543 2 -2 2 -1.10457 0 -2 -0.89543 -2 -2 0 -1.10457 0.89543 -2 2 -2zm8 0c1.10457 0 2 0.89543 2 2 0 1.10457 -0.89543 2 -2 2 -1.10457 0 -2 -0.89543 -2 -2 0 -1.10457 0.89543 -2 2 -2z"
+															android:fillColor="#ffffff"
+															android:strokeLineJoin="round"
+															android:strokeLineCap="round"
+															android:strokeMiterLimit="4" />
+														</group>
+													</group>
+													<path
+														android:pathData="M-695.99988 -64.000221l30.00002 0 0 10.999999 -30.00002 0z"
+														android:fillColor="#cbcbcb"
+														android:strokeLineJoin="round"
+														android:strokeLineCap="round"
+														android:strokeMiterLimit="4" />
+													</group>
+												</group>
+											</vector>
```

Comment:
This will likely have to be reimported using the method I described because I made the experience that Android does not handle groups well.